src/nvim/drawline.c

```diff
@@ -395,6 +398,90 @@ static int get_sign_attrs(buf_T *buf, linenr_T lnum, SignTextAttrs *sattrs, int
   return num_signs;
 }
 
+static void get_statuscol_str(win_T *wp, linenr_T lnum, int row, int startrow, int filler_lines,
+                              int cul_attr, int sign_num_attr, SignTextAttrs *sattrs,
+                              foldinfo_T foldinfo, char_u *extra, statuscol_T *stcp)
+{
+  long relnum;
```

Comment:
Coverity complains about this being uninitialized when passed to `build_statuscol_str()` when `row != startrow` (although `build_statuscol_str()` doesn't actually use the value when `row != startcol`). I guess this just needs an initialization here to silence the warning.